trade-finance-manager-ui/templates/_macros/table-cell-checkbox.njk

```diff
@@ -1,16 +1,28 @@
 {% macro render(params) %}
   {% set checkboxId = params.checkboxId %}
   {% set label = params.label %}
+  {% set checked = params.checked %}
 
   <div class="govuk-checkboxes govuk-checkboxes--small">
     <div class="govuk-checkboxes__item">
-      <input
-        class="govuk-checkboxes__input govuk-!-top-0 table-cell-checkbox-input"
-        id="{{ checkboxId }}"
-        data-cy="table-cell-checkbox--{{ checkboxId }}"
-        name="{{ checkboxId }}"
-        type="checkbox"
-      />
+      {% if checked %}
+        <input
+          class="govuk-checkboxes__input govuk-!-top-0 table-cell-checkbox-input"
+          id="{{ checkboxId }}"
+          data-cy="table-cell-checkbox--{{ checkboxId }}"
+          name="{{ checkboxId }}"
+          type="checkbox"
+          checked
+        />
+      {% else %}
+        <input
+          class="govuk-checkboxes__input govuk-!-top-0 table-cell-checkbox-input"
+          id="{{ checkboxId }}"
+          data-cy="table-cell-checkbox--{{ checkboxId }}"
+          name="{{ checkboxId }}"
+          type="checkbox"
+        />
```

Comment:
I'm hoping we can do this without the else, by doing `checked="{{checked}}"` but I need to check if that works